docs/readmes/orc8r/deploy_using_juju.md

```diff
@@ -6,10 +6,11 @@ hide_title: true
 
 # Deploy Orchestrator using Juju (Beta)
 
-This how-to guide can be used to deploy Magma's Orchestrator on any cloud environment. It contains
-steps to set up a Kubernetes cluster, bootstrap a Juju controller, deploy charmed operators for
-Magma Orchestrator and configure DNS A records. For more information on Charmed Magma, please visit
-the project's [homepage](https://github.com/canonical/charmed-magma).
+This how-to guide can be used to deploy Magma's Orchestrator on any cloud
+environment. It contains steps to set up a Kubernetes cluster, bootstrap
+a Juju controller, deploy charmed operators for Magma Orchestrator
+and configure DNS A records. For more information on Charmed Magma, please
+visit the project's [homepage](https://github.com/canonical/charmed-magma).
```

Comment:
Here I'd point to `https://canonical.github.io/charmed-magma/`

Reply:
Done.